tests/test-upgrade.py

```diff
@@ -106,6 +108,32 @@ def test_upgrade(self):
             except CalledProcessError:
                 print("Will not test the metallb addon")
 
+        if is_ipv6_configured:
```

Comment:
Applying of the launch configuration needs to go when we first install the snap so we make sure it survives the refresh. If you put it here it will get ignored by the refresh.